build/updateLocalConfig.js

```diff
@@ -3,7 +3,7 @@ var _ = require('lodash');
 var execSync = require('child_process').execSync;
 
 const updateLocalConfig = () => {
-  var environment = process.env.SIGNAL_ENV || 'production';
```

Comment:
What is the reasoning here I know SIGN_ENV is mostly prod but do we still not want the option?

Reply:
we always use production, besides we don't need that whole config logic at all